### ISSUE TITLE
BUG: Can't remove all namespaced listeners

### ISSUE THREAD

**Issue by itutto** [leongersen/noUiSlider#1109]
### Remove all listeners in a namespace feature is broken.

```js
noUiSlider.off('.mynamespace'); // this will remove all listeners  not just the ones in the namespace
```

https://github.com/leongersen/noUiSlider/blob/5e026dbdef94db19a8f06d6b2e885854bdc1f0aa/src/nouislider.js#L2097

In the current implementation if only the namespace is provided _(ie: '.namespace')_ to the removeEvent _(or off)_ method it will ignore it and remove all listeners. This feature is stated is supported in the documentation.

#### Related Bug
When removing listeners even the internal listeners are destroyed like aria update.

**Comment by leongersen** [comment]
Thanks for reporting this, that does not look correct. I'll be able to fix and test this somewhere next week, or if you'd be up for submitting a PR, that'd be great!

**Comment by itutto** [comment]
> Thanks for reporting this, that does not look correct. I'll be able to fix and test this somewhere next week, or if you'd be up for submitting a PR, that'd be great!

I'll create a PR today.

**Comment by leongersen** [comment]
I've just merged this and released noUiSlider 14.6.3. Thanks for reporting and contributing, much appreciated! 👍
